src/libraries/System.Formats.Asn1/src/System.Formats.Asn1.csproj

```diff
@@ -0,0 +1,61 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
```

Comment:
I wasn't able to find what this property is meant for, and dind't find anything on Arcade. Was this just a copy-paste property or was it added intentionally?

Reply:
I was also wondering about this.

Reply:
Just a copy.  Looks like S.S.C.Pkcs is the only place it's defined, so that was clearly the project I copied my structure from :smile:.